R/boxr_s3_classes.R

```diff
@@ -120,13 +120,13 @@ as.data.frame.boxr_object_list <- function(x, ...) {
       name                = x$name,
       type                = x$type,
       id                  = x$id,
-      size                = x$size,
+      size                = x$size %|0|% NA,
```

Comment:
If we are specifying `NA`, perhaps we should specify `NA_character_`, `NA_real_`, etc., as the case may be.

Reply:
I don't have any objections to this. The idea is this would add more type specificity, based on what `class(x$size)` already is? So the column types are durable in the event a `NA` value is returned

Reply:
Exactly!